### ISSUE TITLE
ImmutableField false positive

### ISSUE THREAD

**Issue by dgroup** [dgroup/arch4u-pmd#140]
### Summary
| Key            |  Value |
|:---------------|:------|
| Rule           | ImmutableField   |
| arch4u version       | 0.2.0 |

### Your rule definition
```xml
default rule configuration from pmd-java:6.44.0 
```

### Describe the defect or false/positive rule violation
...
```java
@Component
public class TheClass {
    @Value("${app.property}")
    private String property;   // violation is here
}
```

**Comment by dgroup** [comment]
@rultor release tag=`0.2.1`

**Comment by dgroup** [comment]
@rultor release tag=`0.2.2`
